OpenLaTeXStudio/EditorModule/src/latexstudio/editor/LaTeXSettingsPanel.java

```diff
@@ -198,6 +193,11 @@ public void run() {
 
     }
     
+    @SettingListener( setting = ApplicationSettings.Setting.AUTOCOMPLETE_ENABLED )

```

Comment:
It's debatable whether we want to use the same annotation for a setting _listener_ and a setting _changer_. Anyhow the approach should be symmetrical so maybe a SettingWriter annotation is in order.